Web/starter_project_barbies/components/about/HeroSection.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from 'next/image'
+
+const HeroSection = ({}) => {
```

Comment:
Can you use syntax like React.FC to define a component? It is better to make it clear that we are looking at a component